apps/www/_customers/quivr.mdx

```diff
@@ -0,0 +1,80 @@
+---
+name: Quivr
+title: Quivr launch 5,000 Vector databases on Supabase.
+# Use meta_title to add a custom meta title. Otherwise it defaults to '{name} | Supabase Customer Stories':
+# meta_title:
+description: Learn how one of the most popular Generative AI projects uses Supabase as their Vector Store.
+# Use meta_description to add a custom meta description. Otherwise it defaults to {description}:
+meta_description: Learn how one of the most popular Generative AI projects uses Supabase as their Vector Store.
+author: rory_wilding
+author_title: Supabase
+author_url: https://github.com/kiwicopple
+author_image_url: https://avatars2.githubusercontent.com/u/10214025?s=400&u=c6775be2ae667e2acae3ccd347fed62bb3f5b3e7&v=4
+authorURL: https://github.com/kiwicopple
+logo: /images/customers/logos/xendit.png
+logo_inverse: /images/customers/logos/light/xendit.png
+og_image: /images/customers/og/xendit.jpg
+tags:
+  - supabase
+date: '2023-10-05'
+company_url: https://quivr.app
+stats: []
+misc:
+  [
+    { label: 'Use case', text: 'Generative AI' },
+    { label: 'Solutions', text: 'Supabase Vector, Supabase Auth' },
+  ]
+about: "Quivr is an open source 'second brain'. It's like a private ChatGPT, personalized with your own data: you upload your documents and you can then search and ask questions using generative AI."
+---
+
+# Quivr launch 5,000 Vector databases on Supabase
+
+## The challenge: Building a second brain
+
+In May of 2023, [Stan Girard's](https://twitter.com/_StanGirard) started building small prototypes that allowed him to “chat with documents”. After 2 weeks of research, he settled on an idea - build a “second brain” where a user could dump all their digital knowledge (audio, URLs, text, and code) into a vector store and query it with GPT4.
+
+He built the first version in a single afternoon, pushed it to GitHub, and then [tweeted about it](https://twitter.com/_StanGirard/status/1657021618571313155?s=20). One viral tweet later, and [Quivr](https://github.com/StanGirard/quivr) was born.
+
+## Choosing a vector database
+
+A critical piece of the tech stack was the vector store. Stan needed a place to store millions of embeddings. After comparing between Supabase, Pinecone, and Chroma, he settled on [Supabase Vector](https://supabase.com/vector), our open source vector offering for developing AI applications. The decision was driven largely by his familiarity with Postgres, and the tight integration with Vercel.
+
+<Quote img="caryn-marooney.jpeg" caption="Stan Girard, Founder of Quivr.">
```

Comment:
This needs a new avatar

Reply:
@StanGirard let us know which image to use here

Reply:
You can use the one on my lk. Not the best but I'll open a pr when I get a better one 😊 https://www.linkedin.com/in/stanislas-girard